src/goto-programs/read_goto_binary.cpp

```diff
@@ -27,6 +27,8 @@ Module: Read Goto Programs
 #include "elf_reader.h"
 #include "osx_fat_reader.h"
 
+/// Read given goto binary
+/// Does not update config
```

Comment:
Would it be possible to be a bit more explicit? To be frank: this is a commit that has one job, and it doesn't actually do that job. The parameters are not documented, the return value is not explained, failure modes are not covered.